ini.c

```diff
@@ -156,6 +156,14 @@ int ini_parse_stream(ini_reader reader, void* stream, ini_handler handler,
 
         lineno++;
 
+#if INI_USE_STACK
```

Comment:
I think we should do this check whether or not `INI_USE_STACK` is set, right? Because even in non-stack mode we might have read and realloc'd up to `INI_MAX_LINE` bytes.